Library/Homebrew/cmd/install.rb

```diff
@@ -30,7 +30,7 @@ def install_args
         Unless `HOMEBREW_NO_INSTALL_CLEANUP` is set, `brew cleanup` will then be run for
         the installed formulae or, every 30 days, for all formulae.
 
-        Unless `HOMEBREW_NO_INSTALL_UPGRADE` is set, `brew install <formula>` will upgrade <formula> if it
+        Unless `HOMEBREW_NO_INSTALL_UPGRADE` is set, `brew install` <formula> will upgrade <formula> if it
```

Comment:
If you don't want to keep `<formula>` inside the backticks:
```suggestion
        Unless `HOMEBREW_NO_INSTALL_UPGRADE` is set, `brew install foo`, where `foo` is the name of a formula, will upgrade `foo` if it
```

Reply:
I prefer the `<formula>` usage but agree having it in backticks might be nicer

Reply:
Anything in angle brackets becomes [monospaced italic in the Markdown output](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/manpages.rb#L96), but it needs to be outside a backticked sequence for it to output properly, i.e. 
```
`brew install` *`formula`*   # correct
`brew install *`formula`*`   # not correct
```
This is because Markdown allows a monospaced sequence to be italicized in its entirety using delimiters _outside_ the backticks, but not within.